server/models/CollectionJob.js

```diff
@@ -45,6 +45,12 @@ module.exports = function (sequelize, DataTypes) {
             sourceKey: 'testPlanRunId',
             as: 'testPlanRun'
         });
+
+        Model.hasMany(models.CollectionJobTestStatus, {
+            as: 'testStatus',
+            foreignKey: 'collectionJobId',
+            sourceKey: 'id'
+        });
```

Comment:
Do we want the collection job to be deleted when the test plan run is deleted?

Reply:
I debated this myself - we have some recovery potential via the database if not, but we might want to delete them